### DISCUSSION TITLE
HRConvert2 v2.6

### DISCUSSION THREAD

**Discussion by zelon88** [zelon88/HRConvert2#9] [General]
I want to update this app to work with the latest PHP versions, but before I do I want to make a release at the current point so that the Docker repo maintained by @dwaaan is easier to update to the latest version.

<hr /><em>This discussion was created from the release <a href='https://github.com/zelon88/HRConvert2/releases/tag/v2.6'>Release v2.6</a>.</em>

**Comment by dwaaan** [comment]
Docker build is currently failing, not sure why, hope to have it working soon
